# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/asterisk-misdn/makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN chan_misdn-beta-0.1.0/Makefile chan_misdn-beta-0.1.0-new/Makefile
--- chan_misdn-beta-0.1.0/Makefile	2005-04-01 16:32:38.000000000 +0200
+++ chan_misdn-beta-0.1.0-new/Makefile	2005-06-28 21:39:03.000000000 +0200
@@ -24,15 +24,15 @@
 #
 # The Includes are Set appropriatly
 #
-ASTERISKINC=$(ASTERISKSRC)/include
+ASTERISKINC=/usr/include
 
 
 #
 # mISDNuser PATHS
 #
-MISDNUSER=/usr/src/install-misdn/mISDNuser
-MISDNUSERINC=$(MISDNUSER)/include
-MISDNUSERLIB=$(MISDNUSER)/lib
+MISDNUSER=/usr
+MISDNUSERINC=$(MISDNUSER)/include/mISDN
+MISDNUSERLIB=$(MISDNUSER)/lib/mISDN
 
 #
 # mISDNuser Version
@@ -47,10 +47,10 @@
 # If you are using a asterisk version above from stable (v1-0)
 # then comment the following line out (good luck)
 #
-#CFLAGS+=-DASTERISK_STABLE
+CFLAGS+=-DASTERISK_STABLE
 
 
-LINUXROOT=/usr/src/linux-2.6
+LINUXROOT=/usr/src/linux
 
 #
 # Kernel 2.4 ? 
@@ -78,14 +78,14 @@
 
 all: chan_misdn.so
 
-CFLAGS+=-fPIC -I$(ASTERISKINC) -DAST_CONFIG_DIR=\"$(AST_CONFIG_DIR)\" -I$(MISDNUSERINC) -I$(LINUXINC) -I$(MISDNUSER)/i4lnet/ -Wall
-ADDOBJS+=$(MISDNUSER)/i4lnet/libisdnnet.a $(MISDNUSER)/lib/libmISDN.a
+CFLAGS+=-fPIC -I$(ASTERISKINC) -DAST_CONFIG_DIR=\"$(AST_CONFIG_DIR)\" -I$(MISDNUSERINC) -I$(LINUXINC) -I$(MISDNUSERINC)/i4lnet/ -Wall
+ADDOBJS+=$(MISDNUSERLIB)/i4lnet/libisdnnet.a $(MISDNUSERLIB)/libmISDN.a
 
 chan_misdn.so: chan_misdn.o te_lib.o crypt.o ec.o $(ADDOBJS)
 	$(CC) -shared -Xlinker -x -o $@ ec.o te_lib.o crypt.o chan_misdn.o $(ADDOBJS) 
 
 testphone:  testphone.c
-	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSER)/i4lnet/ -I$(LINUXINC)  testphone.c $(MISDNUSERLIB)/libmISDN.a -o testphone
+	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSERINC)/i4lnet/ -I$(LINUXINC)  testphone.c $(MISDNUSERLIB)/libmISDN.a -o testphone
 
 install: chan_misdn.so
 	install -m 644 chan_misdn.so $(INSTALL_MODPATH)/
@@ -107,7 +107,7 @@
 
 
 te_lib.o: te_lib.c te_lib.h isdn_msg.c isdn_msg_parser.c
-	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSER)/i4lnet/ -I$(LINUXINC) $(CFLAGS) -ggdb -c te_lib.c
+	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSERINC)/i4lnet/ -I$(LINUXINC) $(CFLAGS) -ggdb -c te_lib.c
 
 ec.o: ec.c 
 	$(CC) $(CFLAGS) -ggdb -c ec.c
@@ -116,10 +116,10 @@
 	$(CC) $(CFLAGS) -ggdb -c crypt.c
 
 te_test.o: te_test.c
-	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSER)/i4lnet/ -I$(LINUXINC) $(CFLAGS)-ggdb -c te_test.c
+	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSERINC)/i4lnet/ -I$(LINUXINC) $(CFLAGS)-ggdb -c te_test.c
 
 te_test: te_lib.o te_test.o
-	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSER)/i4lnet/ -I$(LINUXINC) $(CFLAGS) te_lib.o te_test.o $(MISDNUSERLIB)/libmISDN.a $(MISDNUSER)/i4lnet/libisdnnet.a -lpthread -ggdb -o te_test
+	$(CC) -I$(MISDNUSERINC) -I$(MISDNUSERINC)/i4lnet/ -I$(LINUXINC) $(CFLAGS) te_lib.o te_test.o $(MISDNUSERLIB)/libmISDN.a $(MISDNUSERLIB)/i4lnet/libisdnnet.a -lpthread -ggdb -o te_test
 #$(MISDNUSERLIB)/libmISDN.a 
 
 clean: 
